TerrorConsole/Assets/_Main/Scripts/Dialogue/DialogueManager.cs

```diff
@@ -0,0 +1,106 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using DG.Tweening;
+using UnityEngine.Rendering;
+using UnityEngine.Serialization;
+using UnityEngine.UI;
+
+namespace TerrorConsole
+{
+    public class DialogueManager : MonoBehaviour
+    {
+        
+        [FormerlySerializedAs("_dialogueCanvas")]
```

Comment:
Remove, this should only be used in critical changes that affect  a massive amount of prefabs